tests/test_routes.py

```diff
@@ -148,3 +148,14 @@ def test_method_not_allowed(self):
         """It should not allow an illegal method call"""
         resp = self.client.put(BASE_URL, json={"not": "today"})
         self.assertEqual(resp.status_code, status.HTTP_405_METHOD_NOT_ALLOWED)
+
+    def test_get_wishlist_list(self):
+        """It should Get a list of Wishlists"""
+        self._create_wishlists(5)
+        resp = self.client.get(
+            BASE_URL,
+            content_type="application/json"
+        )
+        self.assertEqual(resp.status_code, status.HTTP_200_OK)
+        data = resp.get_json()
+        self.assertEqual(len(data), 5)
```

Comment:
One thing that can also be tested is the contents if the contents of the returned list matches the ones you've created.
Other than that, it looks fine.

Reply:
@studentgrad0419 added recommended changes, thanks